src/acvp_cmac.c

```diff
@@ -174,7 +173,7 @@ static ACVP_RESULT acvp_cmac_release_tc (ACVP_CMAC_TC *stc) {
 
 ACVP_RESULT acvp_cmac_kat_handler (ACVP_CTX *ctx, JSON_Object *obj) {
     unsigned int tc_id, msglen, keyLen = 0, keyingOption = 0, maclen, verify = 0;
-    unsigned char *msg = NULL, *key1 = NULL, *key2 = NULL, *key3 = NULL, *mac = NULL;
+    char *msg = NULL, *key1 = NULL, *key2 = NULL, *key3 = NULL, *mac = NULL;
```

Comment:
If you change this to const char * you won;t need to cast further below

Reply:
I think this is outside the scope of this PR. The changes to some of these files were due to changing the relevant utility API, maybe we can have an additional user story to clean up some of these other things? :)

Reply:
Sure, making a new story works 👍